### PR TITLE
Changes to support AGT

### LINKED ISSUE #1063: add support for adcirc-global-test mesh

### PR DESCRIPTION
**Pull request by jasonfleming**
resolves #1063 ... will let it run for a few days before lifting "draft" status

### CODE REVIEW

**Comment by jasonfleming** [comment]
@wwlwpd when I made this PR, GitHub flagged a conflict with master in `config/forcing_defaults.sh` and I resolved it through the GItHub site directly. May not have been a good idea, since it seems that GH merged changes to `master` into the `issue-1063` branch, which is not our usual workflow. Does this merge need to be undone? Please let me know if I need to fix it.

File: asgs_main.sh [hunk 1]
```diff
@@ -81,26 +81,31 @@ checkFileExistence()
...
            curl ${NODALATTRIBUTESURL}/${FNAME}.xz > ${FPATH}/${FNAME}.xz 2> errmsg || warn "$THIS: Failed to download nodal attributes file from ${NODALATTRIBUTESURL}/${FNAME}.xz to ${FPATH}/${FNAME}.xz: `cat errmsg`."
         fi
-        if [[ $FTYPE = "ADCIRC static water level offset data file" ]]; then
+        if [[ $FTYPE == "ADCIRC static water level offset data file" ]]; then
            logMessage "Attempting to download $FTYPE from ${OFFSETURL}/${FNAME}.xz."
             curl ${OFFSETURL}/${FNAME}.xz > ${FPATH}/${FNAME}.xz 2> errmsg || warn "$THIS: Failed to download file from ${OFFSETURL}/${FNAME}.xz to ${FPATH}/${FNAME}.xz: `cat errmsg`."
         fi
+        if [[ $FTYPE == "ADCIRC self attracting earth load tide file" ]]; then
+           logMessage "Attempting to download $FTYPE from ${LOADTIDEURL}/${FNAME}.xz."
+            curl ${LOADTIDEURL}/${FNAME}.xz > ${FPATH}/${FNAME}.xz 2> errmsg || warn "$THIS: Failed to download file from ${LOADTIDEURL}/${FNAME}.xz to ${FPATH}/${FNAME}.xz: `cat errmsg`."
```
Comment (wwlwpd) [inline asgs_main.sh:101]:
```suggestion
           curl ${LOADTIDEURL}/${FNAME}.xz > ${FPATH}/${FNAME}.xz 2> errmsg || warn "$THIS: Failed to download file from ${LOADTIDEURL}/${FNAME}.xz to ${FPATH}/${FNAME}.xz: `cat errmsg`."
```

**Review by wwlwpd — CHANGES_REQUESTED** [review]
Minor change, please accept. otherwise looks good. I can squash and merge. Please confirm. TY!

File: asgs_main.sh (hunk 1 above)
Comment (jasonfleming) [reply asgs_main.sh:101]: Yes I accept! :-)

**Review by wwlwpd — APPROVED** [review]

**Pull request closed without merging**